wasm_utils/src/api_serialization/validation.rs

```diff
@@ -2,6 +2,15 @@ use holochain_core_types::{chain_header::ChainHeader, hash::HashString};
 
 extern crate serde_json;
 
+#[derive(Clone, Serialize, Deserialize, Debug, PartialEq)]
+pub enum ValidationPackage {
```

Comment:
This name is confusing since its the type/definition and not the package itself.